src/lib/comms/sol-lwm2m.c

```diff
@@ -4730,9 +5037,18 @@ sol_lwm2m_client_start(struct sol_lwm2m_client *client)
     struct resource_ctx *res_ctx;
     struct sol_lwm2m_resource res[3];
     int r;
+    static bool setup_access_control = true;
```

Comment:
What happens if in the same program you have two sol_lwm2m_clients with different objects?